adminer/content.md

```diff
@@ -0,0 +1,27 @@
+# Adminer
+
+## What is Adminer?
+
+Adminer (formerly phpMinAdmin) is a full-featured database management tool written in PHP. Conversely to phpMyAdmin, it consist of a single file ready to deploy to the target server. Adminer is available for MySQL, PostgreSQL, SQLite, MS SQL, Oracle, Firebird, SimpleDB, Elasticsearch and MongoDB.
```

Comment:
This is a pretty impressive list of supported databases!  I noticed that the image only installs `pdo_mysql`, `pdo_pgsql`, and `pdo_sqlite` -- does this image still support this full list, or would it need additional extensions to do so? (not necessarily advocating for additional extensions to be installed unless you think it's prudent -- if it doesn't support the others though, which database types this _does_ support probably ought to be noted in this document somewhere)